prawns/governance/src/handlers/democracy.vote.extrinsic.ts

```diff
@@ -21,9 +22,16 @@ export default (network: SubstrateNetwork) =>
       network,
     });
     account.totalProposalVotes = account.totalProposalVotes + 1;
+    account.totalDemocracyReferendaVotes = account.totalDemocracyReferendaVotes + 1;
```

Comment:
In line with dealing with multiple voting by the same account below, should this be a count (or is it worth keeping a separate tally of) 'unique' votes?

Reply:
Good question. It was added as a request from Phala / MCP. Will ask them if they want to include double votes.

Reply:
I have raised the question Kayla who has raised it with Phala. In the interest of moving this forward and given that this is not changing existing behaviour I am going to merge this and will raise a bug/feature (not sure which) off the back of her response.